### ISSUE TITLE
Color not changing in the theme

### ISSUE THREAD

**Issue by itshakeel23** [dkniffin/active_material#30]
I am trying to override primary color in "colors.scss" file but it doesn't reflect my changes when the page is refreshed.

e,g in colors.scss file:
$am-theme-primary: #006CC1 !default;

I applied "blue" color but theme show's "green" color instead.

![screen shot 2016-10-25 at 6 05 11 pm](https://cloud.githubusercontent.com/assets/538055/19688898/8ec07108-9adc-11e6-84ed-7be5c0000223.png)

**Comment by paulitolinhares** [comment]
I'm having the same problem. I'm trying to override the variable on a custom stylesheet but nothing seems to change on the panel.

**Comment by colinambler** [comment]
same here! 

in `active_admin.scss` I have following lines:

```
@import "active_material";

$am-theme-primary: #000 !important;
```

#000 is just test color.   I can't get it to override the variable as it mentions in the docs.  Did anyone break through on this?

**Comment by nhunzaker** [comment]
@colinambler These color values are set as defaults. This means they will respect any existing Sass variables. Any variable assignments added _after_ importing active_material won't be show up.

So instead of:

```
@import "active_material";

$am-theme-primary: #000 !important;
```

Try:

```
$am-theme-primary: #000;

@import "active_material";
```

I also noticed that I didn't make this super clear in our [customization document](https://github.com/vigetlabs/active_material/blob/master/docs/customization.md). I've updated that with more specific information on overriding color variables, and a more comprehensive list of what overrides are available.
